Project.toml

```diff
@@ -7,6 +7,7 @@ version = "0.18.0"
 ClimaParams = "5c42b081-d73a-476f-9059-fd94b934656c"
 DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
 ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
+QuadGK = "1fd47b50-473d-5c70-9696-f719f8f3bcdc"
```

Comment:
Where is QuadGK used? I couldn't find any references

Reply:
In the last integral for the P3 terminal velocities: https://github.com/CliMA/CloudMicrophysics.jl/blob/0945be50c95cecdcefc8561635e4db65305454c4/src/P3Scheme.jl#L621